xdr/ledger_close_meta.go

```
@@ -1,5 +1,18 @@
 package xdr
 
+import "fmt"
+
+func (l LedgerCloseMeta) LedgerHeaderHistoryEntry() LedgerHeaderHistoryEntry {
+	switch l.V {
+	case 0:
+		return l.MustV0().LedgerHeader
+	case 1:
+		return l.MustV1().LedgerHeader
+	default:
+		panic(fmt.Sprintf("Unsupported LedgerCloseMeta.V: %d", l.V))
```

is the service halt unavoidable, there's no way to just throw error instead and let it bubble up to consider how to handle further up stack?